### PR TITLE
Support typescript 4.8

### PR DESCRIPTION
**Pull request by bo4arovpavel1989**
@timocov , hi! Since typescript 4.8 there are some breaking changes 
https://devblogs.microsoft.com/typescript/announcing-typescript-4-8/#decorators-are-placed-on-modifiers-on-typescripts-syntax-trees
could you please review this pull request and release a new (major?) version of the package, if it's ok?

### CODE REVIEW

**Comment by dfilatov** [comment]
@timocov Hi, Evgeny! Could you take a look here?

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   },
   "homepage": "https://github.com/timocov/ts-transformer-minify-privates#readme",
   "peerDependencies": {
-    "typescript": ">=2.9.1"
+    "typescript": ">=4.8.2"
```
Comment (timocov) [inline package.json:27]: is it the first version of when the api you're using is supported? it looks like it is big breaking change in terms of how much major and minor versions we're bumping here

Comment (timocov) [reply package.json:27]: maybe we can play around and create wrappers that will handle this depending on whether an API is supported or not?

**Review by timocov — COMMENTED** [review]
Hi @dfilatov @bo4arovpavel1989, sorry for late reply. Apart from a comment, just curious what it is the reason that you're using this package over https://github.com/timocov/ts-transformer-properties-rename ?

**Comment by bo4arovpavel1989** [comment]
> Hi @dfilatov @bo4arovpavel1989, sorry for late reply. Apart from a comment, just curious what it is the reason that you're using this package over https://github.com/timocov/ts-transformer-properties-rename ?
we use a lot of decorators in the project, but 'ts-transformer-properties-rename'-package would rename private decorated class members as well as not-decorated, which will be broken after mangling by terser

File: package.json (hunk 1 above)
Comment (bo4arovpavel1989) [reply package.json]: > maybe we can play around and create wrappers that will handle this depending on whether an API is supported or not?

done

**Review by timocov — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (timocov) [reply package.json]: Many thanks!

**Comment by timocov** [comment]
> we use a lot of decorators in the project, but 'ts-transformer-properties-rename'-package would rename private decorated class members as well as not-decorated, which will be broken after mangling by terser

@bo4arovpavel1989 can you create an issue for this bug in that repo please (with some examples please as I haven't worked with decorators much and might miss something)? I'd rather to maintain 1 repo than 2 that do pretty similar/the same things :) Or at least we would be aware of that

**Comment by timocov** [comment]
The fix has been released in [v0.5.0](https://github.com/timocov/ts-transformer-minify-privates/releases/tag/v0.5.0). Thanks for contributing @bo4arovpavel1989!

**Pull request merged**